Scarb.toml

```
@@ -19,7 +19,7 @@ description = "Community maintained Cairo and Starknet libraries"
 homepage = "https://github.com/keep-starknet-strange/alexandria/"
 
 [workspace.dependencies]
-starknet = ">=2.4.0"
+starknet = ">=2.5.0"
```

Shouldn't it be this ?
```suggestion
starknet = ">=2.5.3"
```

---

but i can udate it too if you prefer

---

done